packages/graphql-typescript-definitions/test/__snapshots__/cli.test.ts.snap

```diff
@@ -71,16 +71,18 @@ Object {
   },
   "stderr": "",
   "stdout": "
- ERROR  Error parsing 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json':
-
-ENOENT: no such file or directory, open 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json'
-Error: Error parsing 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json':
-
-ENOENT: no such file or directory, open 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json'
+ ERROR  ENOENT: no such file or directory, open 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json'
```

Comment:
I think I'm going to try and repair this in another PR to restore the original error handling.

Reply:
resolved in #37